hypertrace-view-generator/hypertrace-view-generator/src/main/java/org/hypertrace/viewgenerator/generators/SpanEventViewGenerator.java

```diff
@@ -75,8 +69,7 @@ public Schema getSchema() {
                         structuredTrace.getTraceId(),
                         eventMap,
                         childToParentEventIds,
-                        exitSpanToCalleeApiEntrySpanMap,
-                        eventToApiExitCall)
+                        exitSpanToCalleeApiEntrySpanMap)
```

Comment:
Note: Seems like `exitSpanToCalleeApiEntrySpanMap`can also be moved to enricher along with exit call enricher as it's doing similar computation.

Reply:
Yes, seems possible. Will do it separate refactoring pr.